scripts/3.1_assign_primary_feasible_zones.py

```diff
@@ -28,11 +29,15 @@ def main(config_file):
     activity_chains = activity_chains_for_assignment(config)
     logger.info("Activity chains loaded")
 
-    # Filter to a specific day of the week
     logger.info("Filtering activity chains to a specific day of the week")
-    activity_chains = activity_chains[
-        activity_chains["TravDay"] == config.parameters.nts_day_of_week
-    ]
+
+    # Generate random sample of days by household
+    get_chosen_day(config).to_parquet(
```

Comment:
Get a chosen day for each individual to represent a "sample" day given the configured days of the week and whether the household is configured to share a common day.